src/Grid/FieldType/ScheduledCommandExecutionTimeType.php

```diff
@@ -19,28 +19,29 @@ class ScheduledCommandExecutionTimeType implements FieldTypeInterface
      */
     public function render(Field $field, $scheduleCommand, array $options): string
     {
-        if ($scheduleCommand->getCommandEndTime() === null || $scheduleCommand->getExecutedAt() === null) {
+        if ($scheduleCommand->getExecutedAt() === null) {
             return '';
         }
 
-        $time = $scheduleCommand->getCommandEndTime()->getTimestamp() - $scheduleCommand->getExecutedAt()->getTimestamp();
+        $baseDateTime = $scheduleCommand->getCommandEndTime() ?? new \DateTime();
+        $time = $baseDateTime->getTimestamp() - $scheduleCommand->getExecutedAt()->getTimestamp();
 
         if ($time > self::HOUR_IN_SECONDES) {
             $hours = (int) ($time / self::HOUR_IN_SECONDES) . 'h ';
-            $minutes = (($time % self::HOUR_IN_SECONDES) / self::MINUTE_IN_SECONDES) . 'm ';
-            $secondes = (($time % self::HOUR_IN_SECONDES) % self::MINUTE_IN_SECONDES) . 's';
+            $minutes = (int) (($time % self::HOUR_IN_SECONDES) / self::MINUTE_IN_SECONDES) . 'm ';
```

Comment:
That fix this mistake ?
![image](https://user-images.githubusercontent.com/6593252/132680308-13c0bbab-7c40-4fa4-be39-ba3e66960eca.png)

Reply:
yes